LED_data_scrapper/spiders/products_spider.py

```diff
@@ -0,0 +1,51 @@
+""" Module to scrap the data of all the LEDs fromgiven website """
+import scrapy
+from scrapy.loader import ItemLoader
+from LED_data_scrapper.items import ProductItem
+
+class ProductsSpider(scrapy.Spider):
+    """
+    Spider class for Product, here only scrapping LED data
+    """
+
+    name = "products"
+
+    start_urls = ['https://pakistanistores.com/prices/home-appliances/led-tv-prices', ]
+
+    def parse(self, response):
+        # pylint: disable=arguments-differ
+        """ Parsing function """
+        products = response.xpath('//ul[@class="row search-ul"]/\
```

Comment:
Please use rules wherever it's possible.
See this for reference: https://docs.scrapy.org/en/latest/topics/spiders.html